include/CL/sycl/opencl_type.hpp

```diff
@@ -1,80 +1,69 @@
 #ifndef TRISYCL_SYCL_OPENCL_TYPE_HPP
 #define TRISYCL_SYCL_OPENCL_TYPE_HPP
 
-/** \file
-    triSYCL wrapper for OpenCL types
-
-    Joan DOT Thibault AT ens-rennes DOT fr
-
-    This file is distributed under the University of Illinois Open Source
-    License. See LICENSE.TXT for details.
-*/
-
-// Only provide OpenCL interoperability types when OpenCL is in use
-#ifdef TRISYCL_OPENCL
-
-#include <boost/compute.hpp>
```

Comment:
At some point a `cl::sycl::cl_float3` for example has to be connected to an OpenCL `::cl_float3`, with some subtleties such as a `cl::sycl::cl_float3` as actually 4 elements and not 3...

There was recently some similar issue in OpenCL committee https://gitlab.khronos.org/opencl/headers/issues/6